### PR TITLE
TASK5:AWS

### PR DESCRIPTION
**Pull request by veranika-savanchuk**
FE App [link](https://d37shnd892xye0.cloudfront.net/admin/products)
Endpoint for import GET [link]( https://v1i4673f5k.execute-api.eu-west-1.amazonaws.com/dev/import)

### General Tasks

- [x] File serverless.yml contains configuration for importProductsFile function
- [x] The importProductsFile lambda function returns a correct response which can be used to upload a file into the S3 bucket
- [x] Frontend application is integrated with importProductsFile lambda
- [x] The importFileParser lambda function is implemented and serverless.yml contains configuration for the lambda

### Additional (optional) tasks

- [x]  async/await is used in lambda functions
- [x] importProductsFile lambda is covered by unit tests 
- [x] At the end of the stream the lambda function should move the file from the uploaded folder into the parsed folder (move the file means that file should be copied into parsed folder, and then deleted from uploaded folder)

Screen Shot 2020-11-15 at 23 57 44

### CODE REVIEW

**Comment by ihargit** [comment]
Hi, please, check the PUT file functionality from front app.
image
I checked on Windows, there might be an issue with contentType header in PUT 'fetch' request when sending from Windows.
There are two solutions in Discord Task5 topic described, please check.

**Comment by veranika-savanchuk** [comment]
@ihargit Done, could you please try again?

**Comment by ihargit** [comment]
> @ihargit Done, could you please try again?

Thank you, it works! ))

**Comment by veranika-savanchuk** [comment]
@ihargit Cool, thanks a lot for your review 👍

File: import-service/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "main": "handler.js",
+  "scripts": {
+    "deploy": "sls deploy",
+    "test": "jest"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "aws-sdk": "^2.792.0",
+    "babel": "^6.23.0",
```
Comment (aleksei-bulgak-study) [inline import-service/package.json:14]: just curious do you really need this babel dependecies section or may be they can be moved in devDependencies

**Review by aleksei-bulgak-study — APPROVED** [review]

**Pull request merged**